adminutils/adminutils.py

```diff
@@ -188,7 +188,7 @@ async def emoji_add(self, ctx, name: str, url: str, *roles: discord.Role):
             await ctx.tick()
 
     @emoji.command(name="message", aliases=["steal"])
```

Comment:
Since these is second command of "same matter", can you create an group please and move both command there?

I can do it after PR, however

Reply:
Another thought occurred to me. Instead of creating a seperate group, I could make a "steal" command which accepts member, message, or emoji/partialemoji and adds the emoji to the server. This would combine all the functionality into one nice and easy to use command. What do you think?

Reply:
Seems legit, lets try